inst/rmarkdown/templates/gigascience_article/skeleton/skeleton.Rmd

```diff
@@ -0,0 +1,548 @@
+\end{epigraph}
+
+TeX/LaTeX users: Please use OUP’s TeX template and BibTeX stylefile if you use TeX format. During the TeX submission process, please submit your TeX file as the main manuscript file and your bib/bbl file as a dependent file. Please also convert your TeX file into a PDF and submit this PDF as an additional file with the name 'Reference PDF'. This PDF will be used by internal staff as a reference point to check the layout of the article as the author intended. If you have used another template for your manuscript, or if you do not wish to use BibTeX, then please submit your manuscript as a DVI file. We do not recommend converting to RTF.  For all TeX submissions, all relevant editable source must be submitted during the submission process. Failing to submit these source files will cause unnecessary delays in the publication procedures.  [https://academic.oup.com/gigascience/pages/technical_note](https://academic.oup.com/gigascience/pages/technical_note) 
+
+# Notes for using this `rticle` template
+
+## About the GigaScience `rticle` template
+
+The `rticles::gigascience_journal` template is the blended `rmardkdown` and \LaTeX{} template for GigaScience journal manuscript submissions.  If you do not want to include \LaTeX{} commands inline, remove the following line from the YAML header.
+```
+   pandoc_args: raw_tex
+```
+
+## Important preliminaries
+
+- Specify your manuscript's category with the `\papercat{...}` command in the preamble.
+- See the sample code in the YAML header for how author and affiliation information can be specified.
+    - Particularly, note that the first footnote id is reserved for corresponding authors' email addresses.
+
+**Please note that whilst this template provides a preview of the typeset manuscript for submission, it will not necessarily be the final publication layout.**
+
+# Using rmarkdown features with this template
+
+Reference (among others): [https://www.rstudio.com/wp-content/uploads/2015/02/rmarkdown-cheatsheet.pdf](https://www.rstudio.com/wp-content/uploads/2015/02/rmarkdown-cheatsheet.pdf)
+
+## Sectional Headings
+
+You can use `# Section Name` and `## Subsection name` lines to add more sections and subsections to your manuscript. Further sectional levels are provided by `### Subsubsection name`, `#### Paragraph name`, `##### Subparagraph name` lines; e.g.:
+
+```
+  # Header 1
+  section
+  
+  ## Header 2
+  subsection
+  
+  ### Header 3
+  subsubsection
+  
+  #### Header 4
+  paragraph
+  
+  ##### Header 5
+  subparagraph
+  
+  ###### Header 6
+  subsubparagraph
+```
```

Comment:
Could you remove the leading spaces and add a blank line after each header?

Reply:
I would love to.  However, this is an example of what the user is to type, and I want the final document to have these lines indented.  What is the best-practice way to force a triple-backquoted section to be indented in the final document?  

I will add the blank line after each header.

Reply:
Is the indentation only for cosmetic purposes or actually a requirement from the journal? If the former, I hope not to introduce the leading spaces, because these spaces will turn the headers into normal text (paragraphs). Markdown syntax requires no leading spaces before headers.

Reply:
Neither. 

What I am trying to do here is to make it easy for the reader of the generated PDF to see, by the fact that the text is indented, that it is example code.  So, indeed, in this case the intent is to have them be normal text, which is then followed by an example of how they are rendered.

Please look at the generated PDF to understand what I mean if I still have not made myself clear.

Reply:
I hope not to sacrifice technical accuracy in favor of a particular style (indentation). There just should not be spaces before headers. That is the Markdown syntax. If you want a particular style, it should be achieved via styling techniques instead of changing the source (e.g., when you want to right-align text, adding a series of leading spaces to push it to the right margin is often the wrong way; similarly, if you really want to indent text in LaTeX, redefining the particular environment should be the right way to go -- I don't really feel it is worth the time and effort).

Sorry to nitpick here, but accuracy is more important than style to me in this case.